mne_qt_browser/_pg_figure.py

```diff
@@ -1697,10 +1697,10 @@ def __init__(self, main, **kwargs):
 class ProjDialog(_BaseDialog):
     """A dialog to toggle projections."""
 
-    def __init__(self, main, *, name):
+    def __init__(self, main, title='Projectors', **kwargs):
```

Comment:
@juangpc you need to be careful when making large changes including to public functions. One PR should be on one feature / one bug.

Why do you need these changes to address the fix aimed at here? If it's not related to the fix can you revert the unrelated changes and put this in a follow up PR that can be discussed with the other devs?

Reply:
> Why do you need these changes to address the fix aimed at here? 

I don't. Attended here: 0408af1349194cae0016f598f2e85529846728b3